ONSdigital/sdx-seft-consumer-service
# Update sdc-rabbit

## What? and Why?
Update sdc-rabbit library to fix reconnection issues.  Updated other dependencies to keep them current.

## How to test
- Spin it up in sdx-compose and put a spreadsheet through the seft tab of console
- Restart the rabbit container and see if the tries to restart
- Restart it again and make sure the backoff timer resets
- Try and put another spreadsheet after the restart to see it still work

## Checklist
